clang/test/ClangScanDeps/include-tree-with-pch.c

```diff
@@ -3,7 +3,7 @@
 // RUN: split-file %s %t
 // RUN: sed -e "s|DIR|%/t|g" %t/cdb.json.template > %t/cdb.json
 
-// RUN: %clang -x c-header %t/prefix.h -target x86_64-apple-macos12 -o %t/prefix.pch -fdepscan=inline -fdepscan-include-tree -Xclang -fcas-path -Xclang %t/cas
+// RUN: %clang -x c-header %t/prefix.h -target x86_64-apple-macos12 -isysroot %t -o %t/prefix.pch -fdepscan=inline -fdepscan-include-tree -Xclang -fcas-path -Xclang %t/cas
```

Comment:
Was this change necessary?

Reply:
yes, a sysroot is always passed implicitly (through the lit testing configuration) and that doesn't match with the one passed in https://github.com/swiftlang/llvm-project/pull/10493/files#diff-aaa36845ad74ad6019adf78071e73c17d9e5db7080eb49de2605959a742d98faR100

Reply:
Why is that a problem now but wasn't before?

Reply:
The assertion was firing. I _think_ the assertion should always hold unless a build is misconfigured, but I may be missing something.

Reply:
We talked about this offline. Ideally this case wouldn't assert and would be a real diagnostic, but I'm fine with landing this as-is and fixing that in a follow-up.